src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/SplitButtonTests/SplitButtonPage.xaml.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Windows.Input;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+using SplitButton = Microsoft.UI.Xaml.Controls.SplitButton;
+#if HAS_UNO
+using SplitButtonTestHelper = Microsoft.UI.Private.Controls.SplitButtonTestHelper;
+#endif
+using ToggleSplitButton = Microsoft.UI.Xaml.Controls.ToggleSplitButton;
+using ToggleSplitButtonIsCheckedChangedEventArgs = Microsoft.UI.Xaml.Controls.ToggleSplitButtonIsCheckedChangedEventArgs;
+
+namespace UITests.Microsoft_UI_Xaml_Controls.SplitButtonTests
+{
+	/// <summary>
+	/// An empty page that can be used on its own or navigated to within a Frame.
+	/// </summary>
+	[SampleControlInfo("SplitButton", "MUX_Test")]
+	public sealed partial class SplitButtonPage : Page
+    {
+		private int _clickCount = 0;
+		private int _flyoutOpenedCount = 0;
+		private int _flyoutClosedCount = 0;
+
+		public MyCommand TestExecuteCommand;
+		private int _commandExecuteCount = 0;
+
+		private Flyout _placementFlyout;
+
+		public SplitButtonPage()
+		{
+			this.InitializeComponent();
+#if HAS_UNO
+			SplitButtonTestHelper.SimulateTouch = false;
+#endif
+
+			TestExecuteCommand = new MyCommand(this);
+
+			_placementFlyout = new Flyout();
+			_placementFlyout.Placement = FlyoutPlacementMode.Bottom;
+			TextBlock textBlock = new TextBlock();
+			textBlock.Text = "Placement Flyout";
+			_placementFlyout.Content = textBlock;
+
+			ControlStateViewer.ControlType = TestSplitButton.GetType();
+			ControlStateViewer.States = new List<string>
+			{
+				"Normal",
+				"FlyoutOpen",
+				"TouchPressed",
+				"PrimaryPointerOver",
+				"PrimaryPressed",
+				"SecondaryPointerOver",
+				"SecondaryPressed",
+			};
+
+			ToggleControlStateViewer.ControlType = TestSplitButton.GetType();
+			ToggleControlStateViewer.States = new List<string>
+			{
+				"Checked",
+				"CheckedFlyoutOpen",
+				"CheckedTouchPressed",
+				"CheckedPrimaryPointerOver",
+				"CheckedPrimaryPressed",
+				"CheckedSecondaryPointerOver",
+				"CheckedSecondaryPressed",
+			};
+		}
+
+		private void TestSplitButton_Click(object sender, object e)
+		{
+			ClickCountTextBlock.Text = (++_clickCount).ToString();
+		}
+
+		private void TestSplitButtonFlyout_Opened(object sender, object e)
+		{
+			FlyoutOpenedCountTextBlock.Text = (++_flyoutOpenedCount).ToString();
+		}
+
+		private void TestSplitButtonFlyout_Closed(object sender, object e)
+		{
+			FlyoutClosedCountTextBlock.Text = (++_flyoutClosedCount).ToString();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5246693)